src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -123,8 +126,46 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     this.setFocusToTop();
   }
 
+  // Adding validation message to show it as Error Summary
+  generateErrorMessage(fields: CaseField[]){   
+    fields.filter(casefield => !this.caseFieldService.isReadOnly(casefield))
+          .forEach(casefield => {
+            if((this.editForm.controls['data'].get(casefield.id).hasError('required'))){
+              this.validationErrors.push(`${casefield.label} is required #${casefield.id}`);
+              this.editForm.controls['data'].get(casefield.id).markAsTouched();    
+            } else if ((this.editForm.controls['data'].get(casefield.id).hasError('pattern'))){
+              this.validationErrors.push(`${casefield.label} is not valid #${casefield.id}`);
+              this.editForm.controls['data'].get(casefield.id).markAsTouched();              
+            } else if ((this.editForm.controls['data'].get(casefield.id).hasError('minlength'))){
+              this.validationErrors.push(`${casefield.label} required minimum length #${casefield.id}`);
```

Comment:
Please use an object array instead of a string array here. This way you can separate the id from the message.